tools/dumpobj.ml

```diff
@@ -73,80 +73,6 @@ let record_events orig evl =
       Hashtbl.add event_table ev.ev_pos ev)
     evl
 
-<<<<<<< HEAD
-(* Print a structured constant *)
-
-let print_float f =
-  if String.contains f '.'
-  then printf "%s" f
-  else printf "%s." f
-;;
-
-let rec print_struct_const = function
-    Const_base(Const_int i) -> printf "%d" i
-  | Const_base(Const_float f) -> print_float f
-  | Const_base(Const_string (s, _, _)) -> printf "%S" s
-  | Const_immstring s -> printf "%S" s
-  | Const_base(Const_char c) -> printf "%C" c
-  | Const_base(Const_int32 i) -> printf "%ldl" i
-  | Const_base(Const_nativeint i) -> printf "%ndn" i
-  | Const_base(Const_int64 i) -> printf "%LdL" i
-  | Const_block(tag, args) ->
-      printf "<%d>" tag;
-      begin match args with
-        [] -> ()
-      | [a1] ->
-          printf "("; print_struct_const a1; printf ")"
-      | a1::al ->
-          printf "("; print_struct_const a1;
-          List.iter (fun a -> printf ", "; print_struct_const a) al;
-          printf ")"
-      end
-  | Const_float_block a ->
-      printf "[|b ";
-      List.iter (fun f -> print_float f; printf "; ") a;
-      printf "|]"
-  | Const_float_array a ->
-      printf "[|";
-      List.iter (fun f -> print_float f; printf "; ") a;
-      printf "|]"
-
-||||||| merged common ancestors
-(* Print a structured constant *)
-
-let print_float f =
-  if String.contains f '.'
-  then printf "%s" f
-  else printf "%s." f
-;;
-
-let rec print_struct_const = function
-    Const_base(Const_int i) -> printf "%d" i
-  | Const_base(Const_float f) -> print_float f
-  | Const_base(Const_string (s, _, _)) -> printf "%S" s
-  | Const_immstring s -> printf "%S" s
-  | Const_base(Const_char c) -> printf "%C" c
-  | Const_base(Const_int32 i) -> printf "%ldl" i
-  | Const_base(Const_nativeint i) -> printf "%ndn" i
-  | Const_base(Const_int64 i) -> printf "%LdL" i
-  | Const_block(tag, args) ->
-      printf "<%d>" tag;
-      begin match args with
-        [] -> ()
-      | [a1] ->
-          printf "("; print_struct_const a1; printf ")"
-      | a1::al ->
-          printf "("; print_struct_const a1;
-          List.iter (fun a -> printf ", "; print_struct_const a) al;
-          printf ")"
-      end
-  | Const_float_array a ->
-      printf "[|";
-      List.iter (fun f -> print_float f; printf "; ") a;
-      printf "|]"
-
-=======
->>>>>>> ocaml/5.1
```

Comment:
I was confused about this but it looks fine when I cross-reference https://github.com/ocaml-flambda/ocaml-jst/commit/2cff470317783a1483c19abcba4a2de034b5845f#diff-c81880f7e675038d344af33a9e0b1d364fe39f0c31715e9fab07658ba3afc765. The corresponding reading code in `bytecomp/emitcode.ml` is already updated with the corresponding other half of this diff.